.github/workflows/sycl_update_gpu_driver.yml

```
@@ -0,0 +1,49 @@
+name: Update GPU driver
+
+on:
+  schedule:
+    - cron:  '0 3 * * 2'
```

This will allow us to trigger this job manually, just in case.

```suggestion
on:
  schedule:
    - cron:  '0 3 * * 2'
  worflow_dispatch:
```

---

done